java/flight/flight-jdbc-driver/pom.xml

```diff
@@ -220,8 +224,9 @@
                                     <pattern>org.</pattern>
                                     <shadedPattern>cfjd.org.</shadedPattern>
                                     <excludes>
-                                        <exclude>org.apache.arrow.driver.jdbc.**</exclude>
+                                        <exclude>org.apache.arrow.**</exclude>
                                         <exclude>org.slf4j.**</exclude>
+                                        <exclude>org.apache.calcite.**</exclude>
```

Comment:
Excluding these from shade allowed me to attach a debugger while running the shaded jar from DataGrip, solving some problems I couldn't re-create in junit tests.